packages/ui/package.json

```diff
@@ -4,7 +4,8 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "build": "next build",
+    "build-only": "next build",
+    "build": "next build && next export",
```

Comment:
The serverless-amplify-plugin expects `build` command to do everything that's necessary for build which is why we can't call this "build-export" and leave "build" purely as "next build"